tests/test_checker.py

```diff
@@ -20,16 +20,19 @@ def test_version() -> None:
 
 
 @pytest.mark.parametrize("filepath", ALL_TEST_FILES)
-def test_ok_cases_produces_no_errors(filepath: str) -> None:
-    if "ok" in filepath:
-        errors = run_validator_for_test_file(str(filepath))
-
-        assert len(errors) == 0, (str(filepath), errors)
-
+@pytest.mark.parametrize("force_future_annotations", (False, True))
```

Comment:
Can you add separate test cases for this feature? I don't think we need to touch the existing test cases.

Having all test cases combined into a mega test makes it difficult to read.

Reply:
Tell me if that's better now